### PR TITLE
add binding AWS SES

### PR DESCRIPTION
**Pull request by fjvela**
# Description

New binding AWS SES

## Issue reference
- components: https://github.com/dapr/components-contrib/issues/1019
- docs: https://github.com/dapr/docs/issues/1667

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [X] Created/updated tests
* [X] Unit tests passing
* [X] End-to-end tests passing
* [X] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: https://github.com/dapr/docs/issues/1667
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by tanvigour — DISMISSED** [review]
Thanks for doing this

File: go.mod [hunk 1]
```diff
@@ -9,8 +9,9 @@ require (
 	github.com/AdhityaRamadhanus/fasthttpcors v0.0.0-20170121111917-d4c07198763a
 	github.com/PuerkitoBio/purell v1.1.1
 	github.com/agrea/ptr v0.0.0-20180711073057-77a518d99b7b
+	github.com/alibaba/sentinel-golang v1.0.2 // indirect
 	github.com/cenkalti/backoff/v4 v4.1.1
-	github.com/dapr/components-contrib v1.3.0-rc1.0.20210804022438-7496adf1102b
+	github.com/dapr/components-contrib v1.3.0-rc1.0.20210814020225-284d74488f00
```
Comment (CodeMonkeyLeet) [inline go.mod:14]: @artursouza @pkedy It looks like we didn't publish a release tag with the v1.3.0 release for components-contrib ... is that intended?

Comment (yaron2) [reply go.mod:14]: It's not preferable but not the end of the world, as long as this is referencing whatever is the last stable version of contrib that came out with the `1.3` release.

**Comment by yaron2** [comment]
@fjvela please run `go mod tidy` and push the changes. I'll approve soon after.

**Comment by fjvela** [comment]
@yaron2 branch ready to merge.

**Comment by yaron2** [comment]
Thanks!

**Pull request merged**